Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -956,7 +959,8 @@ class BrowserViewController: UIViewController {
         if favoritesViewController == nil {
             let homePanelController = FavoritesViewController(profile: profile,
                                                               fromOverlay: !inline,
-                                                              rewards: rewards)
+                                                              rewards: rewards,
+                                                              background: backgroundDataSource.newBackground())
```

Comment:
We might want to have a mechanism to reload background image when favorites VC is still active.
For example when privacy mode changes, kind of related to https://github.com/brave/brave-ios/issues/2229

Reply:
It does this now.